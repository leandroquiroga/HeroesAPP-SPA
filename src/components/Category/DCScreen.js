import React from 'react';
import { HeroList } from './../Heores/HeroList';

export const DCScreen = () => {
  return (
    <div className='container'>
      <h1 className='text-center'>DC Screen</h1>
      <hr />
      <HeroList publisher='DC Comics' />
    </div>
  );
}