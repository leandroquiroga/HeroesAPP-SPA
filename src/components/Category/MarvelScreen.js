import React from 'react';
import { HeroList } from './../Heores/HeroList';

export const MarvelScreen = () => {
  return (
    <div className='container'>
      <h1 className='text-center'>Marvel Screen</h1>
      <hr />

      <HeroList publisher='Marvel Comics' />
    </div>
  );
};
