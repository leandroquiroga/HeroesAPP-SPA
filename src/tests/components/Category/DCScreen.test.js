import { DCScreen } from './../../../components/Category/DCScreen';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';


describe('Testing on component <DCScreen/>', () => {
  
  test('This test must return a snapshot corretly', () => {
    const wapper = shallow(<DCScreen />);

    expect(toJson(wapper)).toMatchSnapshot();
  })
})