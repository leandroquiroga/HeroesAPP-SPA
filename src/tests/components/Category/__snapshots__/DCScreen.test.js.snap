// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing on component <DCScreen/> This test must return a snapshot corretly 1`] = `
<div
  className="container"
>
  <h1
    className="text-center"
  >
    DC Screen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
